sharing-service/services/sharing/confirmInvite.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+const { getUser } = require('slic-tools/get')
+const { createResponse } = require('slic-tools/response')
+const share = require('./share')
+const { processEvent } = require('slic-tools/event-util')
+const { getSSMValue } = require('./lib/getSSMValue')
+const crypto = require('crypto')
+const { createShareAcceptedEvent } = require('slic-tools/event-dispatcher')
+
+async function main(event) {
+  const { body } = processEvent(event)
+  const { code } = body
+
+  const SECRET = await getSSMValue()
```

Comment:
Do we need to get the secret every time? Also, only use a constant syntax (upper-case) when the values is constant. In this case, it is not, as it is retrieved at runtime.